### PR TITLE
community: Add bio for Razvan Nitu

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: website/docs/community/people.md [hunk 1]
```diff
@@ -4,4 +4,13 @@ title: People
 
 # Project Team
 
-This is content
+## Răzvan Nițu
+
+As part of the OpenEdu (21-COP-0016) team I am charged with defining and maintaining the methodology for creating high quality educational resources.
+
+I am a PhD student at university POLITEHNICA of Bucharest in the field of computer science and engineering.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
I am a PhD student at University POLITEHNICA of Bucharest in the field of computer science and engineering.
```

Comment (razvand) [reply website/docs/community/people.md]: s/in the field of computer science and engineering/in Computer Science and Engineering/

File: website/docs/community/people.md [hunk 2]
```diff
@@ -4,4 +4,13 @@ title: People
...
+My thesis aims at optimiziming memory operations at all possible levels: application, operating system and hardware.
+I am an active contributor to an open source programming language called [D](https://dlang.org/).
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
I am an active open source contributor to [the D programming language](https://dlang.org/), a modern programming language focusing on security and ... TODO.
```

**Review by razvand — CHANGES_REQUESTED** [review]
Some minor comments

File: website/docs/community/people.md [hunk 3]
```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
 
 I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
 
-Apart from computer science, I really enjoy skiing, dancing and cooking.
\ No newline at end of file
+Apart from computer science, I really enjoy skiing, dancing and cooking.
```
Comment (RazvanN7) [inline website/docs/community/people.md]: I don't know what happened here.

Comment (lizababu) [reply website/docs/community/people.md:22]: copycat

File: website/docs/community/people.md [hunk 4]
```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
...
+
+## Răzvan Nițu
+
+![Razvan Nitu](../../static/img/people/liza_babu.jpeg)
+
+As part of the OpenEdu (21-COP-0016) team I am charged with defining and maintaining the methodology for creating high quality educational resources.
+
+I am a PhD student at University POLITEHNICA of Bucharest in Computer Ccience and Engineering.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
I am a PhD student at University POLITEHNICA of Bucharest in Computer Science and Engineering.
```

**Review by lizababu — APPROVED** [review]

File: website/docs/community/people.md (hunk 3 above)
Comment (razvand) [reply website/docs/community/people.md]: What is this?

File: website/docs/community/people.md [hunk 5]
```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
 
 I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
 
-Apart from computer science, I really enjoy skiing, dancing and cooking.
\ No newline at end of file
+Apart from computer science, I really enjoy skiing, dancing and cooking.
+
+## Răzvan Nițu
+
+![Razvan Nitu](../../static/img/people/liza_babu.jpeg)
```
Comment (razvand) [inline website/docs/community/people.md]: This is a link to Liza's picture. Fix it with a link to your picture.

**Review by razvand — CHANGES_REQUESTED** [review]
Why are there two commits in this PR? There should be only one.

**Comment by RazvanN7** [comment]
@razvand fixed.

File: website/docs/community/people.md (hunk 3 above)
Comment (RazvanN7) [reply website/docs/community/people.md]: My guess is Liza's sentence finished with an EOF and I added a newline.

**Review by razvand — CHANGES_REQUESTED** [review]
Please rebase the PR to only have a commit on top of `main`.

Resize your image to about 200-250 size. See the syntax [here](https://raw.githubusercontent.com/open-education-hub/open-education-hub-site/main/website/docs/community/people.md).

**Review by razvand — APPROVED** [review]
Works. Thanks, @RazvanN7

**Pull request merged**